build/push-javadoc-to-gh-pages.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" ]; then
+
+  git clone --quiet --branch=gh-pages https://${GITHUB_OAUTH_TOKEN}@github.com/IBM/java-sdk-core.git gh-pages > /dev/null
```

Comment:
this should be updated to our repo, right?

Reply:
done - will be reflected in the tag update